src/In.ProjectEKA.HipService/Consent/ConsentNotificationController.cs

```diff
@@ -54,19 +54,19 @@ public async Task StoreConsent(ConsentArtefactRepresentation consentArtefact)
             else
             {
                 await consentRepository.UpdateAsync(notification.ConsentId, notification.Status);
-                if (notification.Status == ConsentStatus.REVOKED)
-                {
-                    var patientId = notification.ConsentDetail.Patient.Id;
-                    var cmSuffix = patientId.Split("@")[1];
-                    var gatewayResponse = new GatewayRevokedConsentRepresentation(
-                        Guid.NewGuid(),
-                        DateTime.Now.ToUniversalTime(), 
-                        new ConsentUpdateResponse(ConsentUpdateStatus.OK.ToString(),
-                            notification.ConsentDetail.ConsentId),
-                        null,
-                        new Resp(consentArtefact.RequestId));
-                    await gatewayClient.SendDataToGateway(ConsentOnNotifyPath, gatewayResponse, cmSuffix);
-                }
+                // if (notification.Status == ConsentStatus.REVOKED)
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5748435)